client-management-service-impl/src/main/java/io/mosip/esignet/services/ClientManagementServiceImpl.java

```diff
@@ -134,17 +111,65 @@ public ClientDetailResponse updateOIDCClient(String clientId, ClientDetailUpdate
         clientDetail.setClientAuthMethods(JSONArray.toJSONString(clientDetailUpdateRequest.getClientAuthMethods()));
         clientDetail.setStatus(clientDetailUpdateRequest.getStatus());
         clientDetail.setUpdatedtimes(LocalDateTime.now(ZoneId.of("UTC")));
-        clientDetail = clientDetailRepository.save(clientDetail);
-
-        auditWrapper.logAudit(AuditHelper.getClaimValue(SecurityContextHolder.getContext(), claimName),
-        		Action.OIDC_CLIENT_UPDATE, ActionStatus.SUCCESS, AuditHelper.buildAuditDto(clientId), null);
+        return clientDetail;
+    }
 
+    private ClientDetailResponse toClientDetailResponseDTO(ClientDetail clientDetail) {
```

Comment:
change method name to getClientDetailResponseDTO

Reply:
please check this comment [5a0b048](https://github.com/mosip/esignet/pull/265/commits/5a0b04816d36a5791956c438e0181a232c3f231a)